Scarab/Views/UninstallDependenciesConfirmationWindow.axaml

```diff
@@ -0,0 +1,79 @@
+<Window x:TypeArguments="viewModels:UninstallDependenciesViewModel"
+        xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:ext="clr-namespace:Scarab.Extensions"
+        xmlns:models="clr-namespace:Scarab.Models"
+        xmlns:views="clr-namespace:Scarab.Views"
+        xmlns:customControls="clr-namespace:Scarab.CustomControls"
+        xmlns:viewModels="clr-namespace:Scarab.ViewModels"
+        mc:Ignorable="d" d:DesignWidth="550"
+        x:Class="Scarab.Views.UninstallDependenciesConfirmationWindow"
+        Title="{ext:Localize XAML_UninstallDependencies_Title}"
+        SizeToContent="WidthAndHeight">
+
+  <Window.Styles>
+    <Style Selector="Button">
+      <Setter Property="MinHeight" Value="24" />
+      <Setter Property="MinWidth" Value="80" />
+    </Style>
+  </Window.Styles>
+  <Grid ColumnDefinitions="15,Auto,15,*,15"
+        RowDefinitions="15,Auto,Auto,Auto,15">
+    <Image Grid.Row="1" Grid.Column="1" Margin="0 5 0 0"
```

Comment:
Setting row span as 2 for the images fixes the alignment problems

Reply:
Not quite, this is what happens when you expand the expander. I don't really like that it moves
![image](https://user-images.githubusercontent.com/8714416/236619411-bf660096-45ca-4b16-82b5-d63d1f475796.png)

Reply:
But I also don't really like how it is rn. Maybe create a main grid with 2 columns, left column with only icon right column with another grid for the text. So the text isn't affected by the icons size